sdk/communication/communication-job-router/README.md

```diff
@@ -0,0 +1,396 @@
+
+### Installing
+
+```bash
+npm install @azure/communication-job-router
+```
+
+### Browser support
+
+#### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+
+
+#Tutorial: Route customer support requests to workers using the Azure Communication Services (ACS) Router SDK (WIP)
+In this tutorial, you will learn:
+•	How to create a queue
+•	How to create workers and associate them with a queue
+•	How to route incoming jobs to queues
+
+## Key concepts
+| Name | Description |
+|--|--|
+| RouterClient | This class is needed for router functionality. You instantiate it with your subscription information, and manage workers and queues. |
+| Job | A unit of work that needs to be done. eg. service an incoming call/chat/etc. Jobs will be organized into queues. |
+| Queue | Queues are where jobs wait until they get handled by workers, ordered by priority then enqueue time |
+| Worker | Workers are a resource (e.g. human or bot) that can handle incoming jobs from queues. |
+| Socket | A socket is a property of a worker and represents a worker's capacity to take on new jobs.  When the worker is busy with one or more jobs, the socket is engaged, when a job completes the socket becomes open for new work.  This enables workers to manage the concurrency of jobs being offered to them according to channel. |
+| Channel | The medium upon which a job will be carried out.  In the context of communication scenarios, this includes Voice, Chat, Video, etc. |
+| Classification Policy | A named container for the skills policy, prioritization policy and queue selection policy.  When Contoso creates a new job, a classification policy can be specified that will determine the required skills to fulfill it, its priority and which queue it should be routed to respectively |
+| Distribution Policy | The distribution policy is a container of rules on how jobs are allocated and distributed to workers from a queue. This includes offer time to live, distribution method (e.g. round robin, top (n) available agents, etc) and allocation rules. |
+| Exception Policy | A container of rules that define what action to take when a particular exception occurs.  This container of rules can be associated with one or more queues. |
+
+##Setting Up
+###Install an IDE such as [VSCode](https://code.visualstudio.com/download) or [Webstorm](https://www.jetbrains.com/webstorm/download/) if you haven't (Optional)
+
+###Install NodeJS if you haven't
+LTS or Current version, either is ok.
+https://nodejs.org/en/download/
+
+###Create a new NodeJS Express server
+In a console window (such as cmd, PowerShell, or Bash), create a new folder named RouterQuickStart and use `npx express-generator` to create a new Express app in the current folder. This will create a simple "Hello World" Express project that will listen on port 3000
```

Comment:
```suggestion
### Create a new NodeJS Express server

In a console window (such as cmd, PowerShell, or Bash), create a new folder named `RouterQuickStart` and use `npx express-generator` to create a new Express app in the current folder. This will create a simple "Hello World" Express project that will listen on port `3000`, as follows:

```

We generally like putting these values in code quotes 🤔